### DISCUSSION TITLE
How to install the latest version of podman in Raspberry PI, bullseye

### DISCUSSION THREAD

**Discussion by lumiere11** [containers/podman#20264] [General]
Currently I'm using a Raspberry pi 4 running:
```

Distributor ID: Debian
Description:    Debian GNU/Linux 11 (bullseye)
Release:        11
Codename:       bullseye
```

And I would like to install last stable version 4.6.2.

**Comment by TomSweeneyRedHat** [comment]
@lsm5 do you have your secret decoder ring handy?

**Comment by lsm5** [comment]
@lumiere11 I think your 2 options would be to either build from source or check with the debian maintainers about updating the package. Not much podman upstream can do in this matter.

**Comment by afbjorklund** [comment]
Maybe run Fedora on it (the Pi)

https://docs.fedoraproject.org/en-US/quick-docs/raspberry-pi/
